src/frontend/src/flows/addDevice/manage/addDeviceSuccess.ts

```diff
@@ -0,0 +1,68 @@
+import { html, render } from "lit-html";
+import { mainWindow } from "../../../components/mainWindow";
+import { I18n } from "../../../i18n";
+import { renderPage } from "../../../utils/lit-html";
+import copyJson from "./addDeviceSuccess.json";
+
+export type DeviceAlias = string;
+
+export interface AddDeviceSuccessTemplateProps {
+  deviceAlias: DeviceAlias;
+  onContinue: () => void;
+  i18n: I18n;
+}
```

Comment:
I rather like to have an interface to avoid code duplication because the interface definition of these properties is used in three functions. So in that sense I think it's necessary.

Reply:
Gotcha. We don't really use interfaces anywhere else, would it make sense to stick to a `type` for consistency? 

And for maintainability (and consistency) do you think it'd make sense to keep the type definition in the function and to derive the type alias from the function?

```ts
type TemplateProps = Parameters<typeof addDeviceSuccessTemplate>[0];
```

See also https://github.com/dfinity/internet-identity/blob/23870005b15f5960fbb16494f8a1e0fdcc2cc945/src/frontend/src/flows/recovery/chooseRecoveryMechanism.ts#L72

Reply:
Sure, sounds good. Done.